Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -1212,6 +1216,9 @@ class CombinatorialKalmanFilter {
       // Set accumulatd path to zero before targeting surface
       state.stepping.pathAccumulated = 0.;
 
+      navigator.targetReached(state.navigation, false);
```

Comment:
maybe a short comment why this must be done would be helpful for future developers